packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/editor/EditorToolbarDecorator.kt

```diff
@@ -151,8 +151,10 @@ class EditorToolbarDecorator(
             (editor as? EditorEx)?.permanentHeaderComponent = toolbar
             editor.headerComponent = toolbar
         } ?: run {
-            (editor as? EditorEx)?.permanentHeaderComponent = null
-            editor.headerComponent = null
+            if (editor.headerComponent is MdbJavaEditorToolbar) {
```

Comment:
This runs on every opened editor. This means that if our decorator runs after DataGrip's, we can remove their toolbar. It also can happen with the markdown editor. What we are doing here is, only remove the toolbar _if_ the toolbar that was there is ours.